http-verbs-common/src/main/scala/uk/gov/hmrc/http/HttpTransport.scala

```diff
@@ -101,32 +103,49 @@ trait HttpTransport
     with PostHttpTransport {}
 
 trait CoreGet {
+
   def GET[A](
```

Comment:
Interestingly we provide default parameters for the other http verb calls where as here it's been overloaded. I've updated to provide default parameters for GETs too.

Reply:
Do we want to do the same with the new `VERB(url: URL)` functions? i.e. just one where `headers` has a default param?
Do we know if anyone is overriding the variants that we have removed?